### PR TITLE
RHDEVDOCS-5956: Content creation for GitOps 1.11.2 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

This PR must be CP'ed to GitOps 1.12 and GitOps 1.11 releases.

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5956

Link to docs preview:

https://73151--docspreview.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes#release-notes-for-gitops-title-1-11-2_gitops-release-notes

SME review: @reginapizza 
QE review: @varshab1210 
Peer review: @Srivaralakshmi 

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by varshab1210** [comment]
LGTM

**Review by reginaelyse — APPROVED** [review]
LGTM

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 Good work! Please add the errata updates section as discussed and put [WIP] in the PR description. Please keep the XX and NN parts in the links. Otherwise, the link will throw errors and PR builds won't pass. 

The rest of the content LGTM. Thank you!

**Comment by Srivaralakshmi** [comment]
LGTM. thank you!

File: modules/gitops-release-notes-1-11-2.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+:_mod-docs-content-type: REFERENCE
+[id="release-notes-for-gitops-title-1-11-2_{context}"]
+= Release notes for {gitops-title} 1.11.2
+
+{gitops-title} 1.11.2 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-2_{context}"]
+== Errata updates
+
+[id="rhsa-2024-128925-02-gitops-1-11-2-security-update-advisory_{context}"]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-2.adoc:14]: @Dhruv-Soni11 FYI and FYA:

NOTICE: [RHSA-2024:128925-02](https://errata.engineering.redhat.com/advisory/128925) has changed its advisory name to [RHSA-2024:1346-02](https://errata.engineering.redhat.com/advisory/128925)

Please update the PR and ping me, I will check and merge the RN and publish it. Thank you!

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 PTAL at my comment and let me know once the PR is ready. TY!

File: modules/gitops-release-notes-1-11-2.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-2.adoc]: Fixed it @Srivaralakshmi. PTAL.

File: modules/gitops-release-notes-1-11-2.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+:_mod-docs-content-type: REFERENCE
+[id="release-notes-for-gitops-title-1-11-2_{context}"]
+= Release notes for {gitops-title} 1.11.2
+
+{gitops-title} 1.11.2 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-2_{context}"]
+== Errata updates
+
+[id="rhsa-2024-1346-02-gitops-1-11-2-security-update-advisory_{context}"]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-2.adoc:14]:
```suggestion
[id="rhsa-2024-1346-gitops-1-11-2-security-update-advisory_{context}"]
```

File: modules/gitops-release-notes-1-11-2.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+=== RHSA-2024:1346-02 - {gitops-title} 1.11.2 security update advisory
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-2.adoc:15]:
```suggestion
=== RHSA-2024:1346 - {gitops-title} 1.11.2 security update advisory
```

File: modules/gitops-release-notes-1-11-2.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+Issued: 2023-03-15
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:1346-02[RHSA-2024:1346-02]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-2.adoc:21]:
```suggestion
* link:https://access.redhat.com/errata/RHSA-2024:1346[RHSA-2024:1346]
```

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 PTAL at my comments and revise the PR.

File: modules/gitops-release-notes-1-11-2.adoc (hunk 2 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-2.adoc:14]: Fixed.

File: modules/gitops-release-notes-1-11-2.adoc (hunk 3 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-2.adoc:15]: Fixed.

File: modules/gitops-release-notes-1-11-2.adoc (hunk 4 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-2.adoc:21]: Fixed.

**Pull request merged**